Workflow testing from feature branch to another

Abhi-jith-S opened the pull request:
test

Commit:
VPLAY-10292 : Jira ticket number verification

Reason for change: Jira ticket number verification
Test Procedure: Updated in ticket
Risks: Medium
Priority: P1

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-10292 : Jira ticket number verification

Reason for change:Modified the yml file.Removed checkout to code folder
Test Procedure: Updated in ticket
Risks: Medium
Priority: P1

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=387) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=387) before we can accept your contribution.<br/><hr/>**asasik397** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=387) it.</sub>

Commit:
VPLAY-10292 : Jira ticket number verification

Reason for change: Added on pull_request
Test Procedure: Updated in ticket
Risks: Medium
Priority: P1

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>
